keyboards/gmmk/pro/rev1/ansi/rules.mk

```diff
@@ -13,3 +13,10 @@ AUDIO_ENABLE = no           # Audio output
 ENCODER_ENABLE = yes
 RGB_MATRIX_ENABLE = yes
 RGB_MATRIX_DRIVER = AW20216
+EEPROM_DRIVER = wear_leveling
+WEAR_LEVELING_DRIVER = embedded_flash
+
+LTO_ENABLE = yes
+
+# Enter lower-power sleep mode when on the ChibiOS idle thread
+OPT_DEFS += -DCORTEX_ENABLE_WFI_IDLE=TRUE
```

Comment:
Not sure that this is needed.

Reply:
It's being used on many boards, including almost all Keychron boards, but I am not 100% it works on this one.